daemons/controld/controld_execd.c

```diff
@@ -558,6 +555,10 @@ build_parameter_list(const lrmd_event_data_t *op,
         }
     }
 
+    if (list[0] != '\0') {
```

Comment:
coverity caught your segfault right here - what if list is NULL?

Reply:
Yeah I was rushing to get something out for review before I left yesterday, not a good idea :)

I'll take some time to add unit tests for the add-word functions too